docs/runtimes/create-and-use-functions.ipynb

```diff
@@ -287,7 +287,20 @@
     "    name=\"nuclio\", handler=\"nuclio_func:nuclio_handler\",\n",
     "    image=\"mlrun/mlrun\", kind=\"nuclio\", with_repo=True,\n",
     ")\n",
-    "```"
+    "```\n",
+    "#### Load code at runtime using a non-default source\n",
+    "If your project already has a default source, you can still use a different source at runtime or build. Based on the previous example:\n",
+    "\n",
+    "```python\n",
+    "archive_url = \"https://s3.us-east-1.wasabisys.com/iguazio/project-archive/project-archive.zip\"\n",
+    "project.set_source(source=archive_url, pull_at_runtime=True)\n",
+    "\n",
+    "fn = project.set_function(\n",
+    "    name=\"nuclio\", handler=\"nuclio_func:nuclio_handler\",\n",
+    "    image=\"mlrun/mlrun\", kind=\"nuclio\", with_repo=True,\n",
+    ").with_source_archive(...)\n",
```

Comment:
I think for clarity, it would be better to put this in a separate line so it's easier to spot the difference.
Replace the last part with a new line:
```
 fn.with_source_archive("https://s3.us-east-1.wasabisys.com/some/other/archive.zip")
```